Source/WebCore/style/UserAgentStyle.cpp

```diff
@@ -176,12 +176,12 @@ void UserAgentStyle::initDefaultStyleSheet()
     mediaQueryStyleSheet = &StyleSheetContents::create(CSSParserContext(UASheetMode)).leakRef();
 
     // Strict-mode rules.
-    String defaultRules = String(StringImpl::createWithoutCopying(htmlUserAgentStyleSheet, sizeof(htmlUserAgentStyleSheet))) + RenderTheme::singleton().extraDefaultStyleSheet();
+    String defaultRules = String(StringImpl::createWithoutCopying(htmlUserAgentStyleSheet)) + RenderTheme::singleton().extraDefaultStyleSheet();
     defaultStyleSheet = parseUASheet(defaultRules);
     addToDefaultStyle(*defaultStyleSheet);
 
     // Quirks-mode rules.
-    String quirksRules = String(StringImpl::createWithoutCopying(quirksUserAgentStyleSheet, sizeof(quirksUserAgentStyleSheet))) + RenderTheme::singleton().extraQuirksStyleSheet();
+    String quirksRules = String(StringImpl::createWithoutCopying(quirksUserAgentStyleSheet)) + RenderTheme::singleton().extraQuirksStyleSheet();
```

Comment:
I guess this might be good when `extraQuirksStyleSheet` is empty since we would never copy the characters. But if it’s non-empty then we will copy the characters and `makeString` would do a better job since we could pass it a span and we wouldn’t have to allocate and destroy a temporary `StringImpl` for the user agent style sheet.